classes/07class/exercises/c07-serverless02/alanlima/Makefile

```diff
@@ -0,0 +1,86 @@
+include .env
```

Comment:
Check comments on Makefile for exercises `c06-serverless01` and `c07-serverless01`